README.md

```diff
@@ -96,7 +96,19 @@ FAR CRs are deleted by NHC after it sees the Node is healthy again.
 
 #### Example CR
 
-The FAR, `FenceAgentsRemediation`, CR is created by the admin and is used to trigger the fence-agent on a specific node. The CR includes an *agent* field for the fence-agent name, *sharedparameters* field with all the shared, not specific to a node, parameters, and *nodeparameters* field to specify the parameters for the fenced node.
+The FAR CR (CustomResource), created by the admin, is used to trigger the fence-agent on a specific node.
+
+The CR includes the following parameters:
+* `agent` - the fence-agent name
+* `sharedparameters` - all the shared, not specific to a node, parameters
+* `nodeparameters` - the parameters for the fenced node
+ 
+Optionally, the CR can include further parameters to specify how the operator should retry the fence-agent command in case of failure:
```

Comment:
I wouldn't say that the `the CR can include` as it will always include these fields (by default or by manually adding them).

I would even append these fields/parameters on the above bullet list and combine this line with [line 101](https://github.com/medik8s/fence-agents-remediation/pull/121/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R101)

Reply:
It makes sense, maybe just indicating that they are optional with default values

Reply:
All the fields are optional, except _agent_, so they are not different than others. What is "unique" is that they have default values that other parameters don't have.